Formula/questdb.rb

```diff
@@ -18,6 +18,7 @@ def install
     rm_rf "questdb.exe"
     libexec.install Dir["*"]
     (bin/"questdb").write_env_script libexec/"questdb.sh", Language::Java.overridable_java_home_env("11")
+    inreplace libexec/"questdb.sh", "/usr/local/var/questdb", opt_prefix
```

Comment:
```suggestion
    inreplace libexec/"questdb.sh", "/usr/local/var/questdb", opt_prefix/"var/questdb"
```

Reply:
Thanks for spotting this. I need to learn more about where various variables point to!

Reply:
I've not looked at the actual file, but it sounds like it should be:

```suggestion
    inreplace libexec/"questdb.sh", "/usr/local/var/questdb", var/"questdb"
```

https://github.com/Homebrew/brew/blob/c65bb5d12981267fada2c5644a02bf814a639188/Library/Homebrew/formula.rb#L877

Reply:
ah right, I looked for `opt_var` and couldn't find it.

Reply:
that's a good suggestion, thanks!